prow/config.yaml

```diff
@@ -100,7 +100,7 @@ tide:
     reviewApprovedRequired: true
   - repos:
     - kubernetes-sigs/testing_frameworks
-    - kubernetes-sigs/scheduling_poseidon
+    - kubernetes-sigs/poseidon
```

Comment:
not `kubernetes-sigs/sig-scheduling_poseidon`? :-)
https://github.com/kubernetes/community/blob/master/kubernetes-repositories.md#rules-1
:man_shrugging:

Reply:
yes, we created it like that then we re-named it as `poseidon` which is simpler.